nautobot_chatops_ipfabric/worker.py

```diff
@@ -11,49 +11,118 @@
 from nautobot_chatops.workers import subcommand_of, handle_subcommands
 from netutils.ip import is_ip
 from netutils.mac import is_valid_mac
-from ipfabric_diagrams import IPFDiagram, Unicast
-from ipfabric import IPFClient
+from ipfabric_diagrams import Unicast
 
-from .ipfabric import IpFabric, LAST
+from .ipfabric_wrapper import (
+    IpFabric,
+    LAST,
+    DEFAULT_PAGE_LIMIT,
+)
 from .context import get_context, set_context
+from .utils import parse_hosts
 
 BASE_CMD = "ipfabric"
 IPFABRIC_LOGO_PATH = "ipfabric/ipfabric_logo.png"
 IPFABRIC_LOGO_ALT = "IPFabric Logo"
-
+CHATOPS_IPFABRIC = "nautobot_chatops_ipfabric"
+EMPTY = "(empty)"
+
+# URLs
+INVENTORY_DEVICES_URL = "tables/inventory/devices"
+INTERFACE_LOAD_URL = "tables/interfaces/load"
+INTERFACE_ERRORS_URL = "tables/interfaces/errors/bidirectional"
+INTERFACE_DROPS_URL = "tables/interfaces/drops/bidirectional"
+BGP_NEIGHBORS_URL = "tables/routing/protocols/bgp/neighbors"
+WIRELESS_SSID_URL = "tables/wireless/radio"
+WIRELESS_CLIENT_URL = "tables/wireless/clients"
+ADDRESSING_HOSTS_URL = "tables/addressing/hosts"
+
+# COLUMNS
+INVENTORY_COLUMNS = [
```

Comment:
I would prefer having this in the wrapper class

Reply:
agreed